src/saturn_engine/worker/topics/rabbitmq.py

```diff
@@ -72,6 +72,7 @@ class Options:
         log_above_size: t.Optional[int] = None
         max_publish_concurrency: int = 8
         max_retry: int | None = None
+        dead_letter_queue: str | None = None
```

Comment:
I would suggest two changes that would cover everything we need:

1. Add a `queue_arguments: dict`, that we can just pass as-is to `declare_queue`. This way we won't have to open  a new PR each time we want something new.
2. Add an `exchange_name: Optional[str]` that we can use to declare and bind an exchange is not `None`?

This way we don't need to create a new class

Reply:
Might as well make it `exchange: Optional[ExchangeOptions]` so we support the few exchange parameters and can do fanout with this PR as well.

```
@dataclass
class ExchangeOptions:
  name: str
  type: ExchangeType
  ...
```

Reply:
But we still need to knew its a dead letter queue for the arguments no?

Reply:
(All the options: https://aio-pika.readthedocs.io/en/latest/apidoc.html#aio_pika.Channel.declare_exchange)

Seems like `durable` we need to set `True`